jmeter-java-dsl-graphql/src/test/java/us/abstracta/jmeter/javadsl/graphql/DslGraphqlSamplerTest.java

```diff
@@ -137,6 +138,41 @@ public void shouldSendGraphqlQueryToServerWhenGraphqlSamplerWithHttpGet()
     ).run();
     verify(getRequestedFor(anyUrl()).withQueryParam("query", equalTo(QUERY)));
   }
+  @Test
+  public void shouldThrowExceptionForInvalidVariable() {
+    DslGraphqlSampler sampler = graphqlSampler("http://localhost", "{ user(id: 1){ name } }");
+    assertThrows(IllegalArgumentException.class, () -> sampler.variable("invalidVar", new Object()));
+  }
+
+  @Test
```

Comment:
Any particular reason to add this test? There is no particular logic in sampler for handling length of operation name (as it happens in rest of the code for long strings), so I tend to think this test is not needed, and would be better to remove it.